frontend/src/components/timergroup/timergroup.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
```

Comment:
As above, plz change timergroup.jsx to timerGroup.jsx and timergroup.module.css to timerGroup.module.css :)